resources/dicts/patrols/general/medcat.json

```diff
@@ -822,7 +887,8 @@
     "exp": 20,
     "success_text": [
         "r_c makes a humming sound, and asks p_l if they remember a particular story the elders are fond of. p_l nods, and r_c asks what they think the moral is. It eventually turns into talking about multiple old stories, which certainly makes for interesting conversation, and it's all too soon that they have their mouths too full of herbs to continue.",
-        "r_c, deep in thought, asks p_l if they remember a particular story the elders are fond of. p_l swishes their tail, and asks why. It's just something they've been dwelling on recently, r_c explains, and they wanted to know what p_l thought of it. p_l nods, and says they'll answer at camp - but they've reached the herbs, and have to pick them now."
+        "r_c, deep in thought, asks p_l if they remember a particular story the elders are fond of. p_l swishes their tail, and asks why. It's just something they've been dwelling on recently, r_c explains, and they wanted to know what p_l thought of it. p_l nods, and says they'll answer at camp - but they've reached the herbs, and have to pick them now.",
+        "r_c asks s_c if they remember a particular story the elders are fond of. s_c nods eagerly, and launches into recanting it. r_c is pleasantly surprised at how readily s_c recites the tale, listening to them talk in content silence. s_c's thrilled to share their knowledge, and it's all too soon that they have their jaws too full of herbs to continue."
```

Comment:
didja forget to add the win skills list on this one?

Reply:
man. i sure did